apps/linode-marketplace-influxdb/roles/post/templates/motd.j2

```diff
@@ -0,0 +1,7 @@
+*********************************************************
+Akamai Connected Cloud InfluxDB Marketplace App
+App URL: https://{{ _domain }}
+Credentials File: /root/.credentials
```

Comment:
Change cred file location to: `Credentials File: /home/{{ username }}/.credentials`

Reply:
corrected in [0b8e27b](https://github.com/akamai-compute-marketplace/marketplace-apps/pull/120/commits/0b8e27bb9b6641c2719566c7c337299997d95b54)